docs/pages/tutorial/onchain-chess/2-move.md

```diff
@@ -4,20 +4,19 @@
 
 ```c
     #[abi(embed_v0)]
-    impl PlayerActionsImpl of IActions<ContractState> {
+    impl IActionsImpl of IActions<ContractState> {
```

Comment:
I can't comment on the line above, should be `rust` and not `c`? Or `c` is better in this situation?

Reply:
It also work with `rust`, but due to actual code is `cairo`, in some cases the IDE highlights it somewhat oddly, resembling a blend of comments—a peculiar behavior. In some situtations, when using C, the highlights appear cleaner.